imap_processing/run_processing.py

```diff
@@ -0,0 +1,122 @@
+"""Serves as a command line utility to run the processing for a specific
+instrument and data level.
+
+Use
+---
+    python run_processing.py <instrument> <data_level>
+"""
+
+import argparse
+
+from imap_processing import instruments, processing_levels
+
+
+def _parse_args():
+    """Parse the command line arguments.
+
+    Returns
+    -------
+    args : argparse.Namespace
+        An object containing the parsed arguments and their values
+    """
+    description = (
+        "This command line program invokes the processing pipeline "
+        "for a specific instrument and data level. Example usage: "
+        '"python run_processing swe l1a".'
+    )
+
+    instrument_help = (
+        "The instrument to process. Acceptable values are: " '"swe", and "codice".'
+    )
+    level_help = (
+        "The data level to process. Acceptable values are: "
+        '"l0", "l1a", "l1b", "l1c", and "l2".'
+    )
```

Comment:
Are you able to make this dependent on the different levels depending on what was specified as the instrument?